src/sentry/workflow_engine/models/data_condition.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, region_silo_model, sane_repr
+
+from .data_condition_group import DataConditionGroup
+
+
+@region_silo_model
+class DataCondition(DefaultFieldsModel):
+    """
+    A data condition is a way to specify a logic condition, if the condition is met, the condition_result is returned.
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
+    __repr__ = sane_repr("type", "condition")
+
+    condition = models.CharField(max_length=200)
+    threshold = models.FloatField()
+    condition_result = models.JSONField()
+    type = models.CharField(max_length=200)
+
+    group = models.ForeignKey(
```

Comment:
not a huge fan of `group` here, any other recommendations / suggestions? (I'm nervous about name collisions with `Group` for issues.

Reply:
What about `condition_group` or `condition_clause` 🤷

Reply:
👍 i'll go with `condition_group` i feel like that's different enough from `issues` but close to this. thanks!